astropy_helpers/utils.py

```diff
@@ -293,12 +270,8 @@ def resolve_name(name):
     return ret
 
 
-if sys.version_info[0] >= 3:
-    def iteritems(dictionary):
-        return dictionary.items()
-else:
-    def iteritems(dictionary):
-        return dictionary.iteritems()
+def iteritems(dictionary):
+    return dictionary.items()
```

Comment:
This can be removed, and uses of this can be changed to ``.items()`` instead

Reply:
Good catch, as far as I see this is not even used anywhere.